CreeDictionary/DatabaseManager/xml_entry_lemma_finder.py

```diff
@@ -3,20 +3,65 @@
 """
 from collections import defaultdict
 from itertools import chain
-from typing import List, Dict, Tuple, Set
+from string import Template
+from typing import List, Dict, Tuple, Set, Optional, Any
 
 from colorama import Fore, init
+from typing_extensions import Literal
 
 import utils
 from DatabaseManager.log import DatabaseManagerLogger
 from DatabaseManager.xml_consistency_checker import does_lc_match_xml_entry
-from utils.types import FSTLemma, ConcatAnalysis
-from utils.enums import SimpleLC
 from shared import strict_analyzer
+from utils import shared_res_dir, SimpleLC, ConcatAnalysis, FSTLemma
+import csv
 
 init()  # for windows compatibility
 
 
+class DefaultLemmaPicker:
+    """
+    When two analyses have the same looking lemma and word class, this helps choose the preferred lemma
+
+    i.e. It helps solve this question: "Which one of maskwa+N+A+Sg and maskwa+N+A+Obv is the lemma?"
+    """
+
+    def __init__(self, language: Literal["crk", "gunaha"]):
```

Comment:
```suggestion
    def __init__(self, language: Literal["crk", "srs"]):
```

If we're using the ISO 639 code for Tsuut'ina.

| Language | ISO 639-3 | App Name |
| - | - | - |
| Plains Cree | crk | itwêwina |
| Tsuut'ina | srs | Gúnahà |

Reply:
yyeaeeah thanks